src/sink.rs

```diff
@@ -79,7 +80,7 @@ impl Sink {
     ///
     /// No effect if not paused.
     #[inline]
-    pub fn play(&self) {
+    pub fn resume(&self) {
```

Comment:
Please add this as a separate method and put `#[deprecated]` on `play`. I don't want to break the API randomly like this.